packages/next/server/node-web-streams-helper.ts

```diff
@@ -207,9 +207,11 @@ export function createBufferedTransformStream(): TransformStream<
     return pendingFlush
   }
 
+  const textDecoder = new TextDecoder();
```

Comment:
We need to keep the same TextEncoder instance until the stream ends, or else the steam option won't work.